sources/Interop/Windows/other/helper-types/HANDLE.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+using System;
+
+namespace TerraFX.Interop
+{
+    public unsafe partial struct HANDLE : IEquatable<HANDLE>
+    {
+        public readonly nint Value;
+
+        public HANDLE(nint value)
+        {
+            Value = value;
+        }
+
+        public static HANDLE INVALID_HANDLE_VALUE => new HANDLE(-1);
```

Comment:
on a brevity note, I might expect these to just be called `INVALID` or `INVALID_VALUE`, although `INVALID_HANDLE_VALUE` is more 1:1 w/ the headers

Reply:
Good point. Fixed.

Reply:
`HANDLE.INVALID` or `HANDLE.INVALID_VALUE` seems to type and read better than `HANDLE.INVALID_HANDLE_VALUE` which feels redundant